sourcecode/hub/resources/views/content/show.blade.php

```diff
@@ -1,27 +1,92 @@
+                        <x-icon name="pencil" class="me-1" />
+                    </button>
+                    <button type="button" class="btn btn-secondary">
+                        {{ trans('messages.use-resource')}}
+                    </button>
+                </div>
+                <div>
+                    <button type="button" class="btn btn-secondary" data-bs-toggle="modal" data-bs-target="#deletionModal">
+                        {{ trans('messages.delete')}}
+                        <x-icon name="trash" class="me-1" />
+                    </button>
+                </div>
+            </div>
+        </div>
+        <hr/>
+
+        <div class="row justify-content-around mt-4">
+            <ul class="nav nav-underline d-flex w-100 justify-content-between">
+                <li class="nav-item">
+                    <a class="nav-link active" aria-current="page" href="#">{{ trans('messages.licensing')}}</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#">{{ trans('messages.user-statistics')}} <x-icon name="graph-up-arrow" class="text-black"/></a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#">{{ trans('messages.using-resource-on-lms')}}</a>
+                </li>
+            </ul>
+        </div>
+    </div>
 
     <x-slot:sidebar>
         @can('edit', $content)
             <section>
-                <h2 class="fs-5">{{ trans('messages.version-history') }}</h2>
-
+                <h2 class="fs-5">{{ trans('messages.version-history') }}
+                    <span data-bs-toggle="tooltip" data-bs-placement="top" title="{{ trans('messages.version-history-tooltip')}}">
+                        <x-icon name="info-circle" class="ms-2" />
+                    </span>
+                </h2>
                 <ul class="p-0">
-                    @foreach ($content->versions as $version)
-                        <li class="d-block p-1 mb-1 {{ $version->published ? ($content?->latestPublishedVersion->is($version) ? 'bg-success text-bg-success' : 'bg-success-subtle'): 'bg-danger-subtle' }}">
-                            {{ $version->created_at }}
-                            @if ($content?->latestPublishedVersion->is($version))
-                                <span class="d-block">{{ trans('messages.published') }}</span>
-                            @endif
-                        </li>
+                    @foreach ($content->versions->sortByDesc('created_at')->take(3) as $index => $version)
+                        <x-version-details :version="$version" :index="$index" :loop="$loop" />
                     @endforeach
                 </ul>
+
+                @if(count($content->versions) > 3)
+                    <button class="btn btn-link d-flex align-items-center" type="button" data-bs-toggle="collapse" data-bs-target="#collapseVersions" aria-expanded="false" aria-label="collapseVersions">
```

Comment:
`arial-label` will be read by screen readers and should be translatable.
`aria-labelledby` is an id that points to the text that the screen-reader will read.
Using `aria-label` the translatable text should be a short text describing what the button does i.e. "Toggle listing of all versions"